CHANGELOG.md

```diff
@@ -4,7 +4,37 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
-## [v2.0.1](https://github.com/voxpupuli/puppet-openvmtools/tree/v2.0.1) (2020-01-24)
+## [v2.1.0](https://github.com/voxpupuli/puppet-openvmtools/tree/v2.1.0) (2021-05-10)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-openvmtools/compare/v2.0.1...v2.1.0)
+
+**Breaking changes:**
+
+- Require puppet-epel over stahnma-epel [\#29](https://github.com/voxpupuli/puppet-openvmtools/pull/29) ([dhoppe](https://github.com/dhoppe))
```

Comment:
This is marked as a breaking change but this isn't a major version bump. That's a bit odd to me.

Reply:
I just ran the github changelog generator. If that shall be changed either the label of that PR needs to change (i do not have permissions to do it) or we need to adapt the rake task to not identify that label as "breaking"...

Reply:
It's complicated whether it really is breaking or not. I can see the argument for both.

Reply:
Check here: https://github.com/voxpupuli/puppet-nodejs/pull/419 ...there it was not labeled with "backwards-incompatible" (causing the "breaking-change") and just with "doc" and it was merged by yourself. Maybe that helps on the decision?

Reply:
@ekohl Can we get progress here?